steps/image-build/step.yaml

```diff
@@ -7,18 +7,18 @@ kind: Step
 
 # The name of the action. Required. Must be exactly the name of the directory
 # containing the action.
-name: kaniko-step-image-build
+name: image-build
 
 # The version of the action. Required. Must be an integer. If specified in the
 # directory name, must be exactly the version in the directory name.
 version: 3
 
 # High-level phrase describing what this action does. Required.
-summary: Build image
+summary: Build Image

```

Comment:
Aren't these usually phrases? "Build and push a Docker image" would be more conventional I think